apps/twap/src/app/plugins/env.ts

```diff
@@ -0,0 +1,53 @@
+import fp from 'fastify-plugin';
+import fastifyEnv from '@fastify/env';
+
+const schema = {
+  type: 'object',
+  required: [
+    'DATABASE_NAME',
+    'DATABASE_USERNAME',
+    'DATABASE_PASSWORD',
+    'DATABASE_HOST',
+    'DATABASE_PORT',
+  ],
+  properties: {
+    DATABASE_NAME: {
+      type: 'string',
+    },
+    DATABASE_HOST: {
+      type: 'string',
+    },
+    DATABASE_PORT: {
+      type: 'number',
+    },
+    DATABASE_USERNAME: {
+      type: 'string',
+    },
+    DATABASE_PASSWORD: {
+      type: 'string',
+    },
+  },
+};
+
+export default fp(async (fastify, opts) => {
```

Comment:
What does `fp` stands for?
A special name in the freamework?

Reply:
fastify plugin. it is a default export you can name it as you wish, but conventionally this is the naming used.

Reply:
No need, just wanted to understand where it came from.